gulpfile.js

```diff
@@ -72,28 +75,16 @@ function test() {
 gulp.task('build-test', ['build'], test);
 gulp.task('test', test);
 
-function runCustomVerification(pathInTools, errorMessage, cb) {
-    var checkProcess = child_process.fork(path.join(__dirname, "tools", pathInTools),
-        {
-            cwd: path.resolve(__dirname, "src"),
-            stdio: "inherit"
-        });
-    checkProcess.on("error", cb);
-    checkProcess.on("exit", function (code, signal) {
-        if (code || signal) {
-            cb(new Error(errorMessage));
-        } else {
-            cb();
-        }
-    });
-}
-
-gulp.task('checkImports', function (cb) {
-    runCustomVerification("checkCasing.js", "Mismatches found in import casing", cb);
+gulp.task('check-imports', function (cb) {
+    var tsProject = ts.createProject('tsconfig.json');
+    return tsProject.src()
+        .pipe(imports());
 });
 
-gulp.task('checkCopyright', function (cb) {
-    runCustomVerification("checkCopyright.js", "Some source code files don't have the expected copyright notice", cb);
+gulp.task('check-copyright', function (cb) {
+    var tsProject = ts.createProject('tsconfig.json');
```

Comment:
Will this verify the copyright of all source files? We want to run this for the gulpfile and other .js files that we have in the repo.